connection/kafka_http_sideline_conn.go

```diff
@@ -0,0 +1,57 @@
+package connection
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/Shopify/sarama"
+	"github.com/flipkart-incubator/go-dmux/core"
+	sink "github.com/flipkart-incubator/go-dmux/http"
+	source "github.com/flipkart-incubator/go-dmux/kafka"
+	"github.com/flipkart-incubator/go-dmux/offset_monitor"
+	"log"
+	"os"
+)
+
+// **************** CONFIG ***********
+
+//KafkaHTTPWithSidelineConn struct to abstract this connections Run
+type KafkaHTTPWithSidelineConn struct {
+	EnableDebugLog bool
+	Conf           interface{}
+	SidelineImpl   interface{}
+}
+
+func (c *KafkaHTTPWithSidelineConn) getConfiguration() *KafkaHTTPConnConfig {
+	data, _ := json.Marshal(c.Conf)
+	var config *KafkaHTTPConnConfig
+	json.Unmarshal(data, &config)
+	return config
+}
+
+//Run method to start this Connection from source to sink
+func (c *KafkaHTTPWithSidelineConn) Run() {
+	conf := c.getConfiguration()
+	fmt.Println("starting go-dmux with conf", conf)
```

Comment:
any reason to mix fmt and log statements? Since there is no separate logger defined, both of it will come to stdout, hence its makes more sense one of them. Personally I prefer to use `log` since it logs with timestamp by default, though that can be configured.

Reply:
it is a miss from my side, will change to log in all the places.